include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -393,7 +393,7 @@ ERROR(error_creating_remark_serializer,none,
       "error while creating remark serializer: '%0'", (StringRef))
 
 REMARK(interface_file_lock_failure,none,
-      "could not acquire lock file for module interface '%0'", (StringRef))
+      "building module interface without lock file", ())
```

Comment:
That's much less scary, I like it!

Would it be useful to say what is the drawback here, like "this may oversubscribe the CPU"? But I'm not sure if it would be reassuring knowing why me remark on lock files or bring more confusion tbh.

Reply:
yeah, I don't think this additional information offers much value to the users.